.gitignore

```diff
@@ -1,6 +1,3 @@
+.phpunit.result.cache
 composer.lock
-phpunit.xml
```

Comment:
phpunit.xml should be excluded. 

phpunit.xml.dist is the example config. phpunit.xml is used localy in the same way as .env with .env.dist